pages/_app.js

```diff
@@ -9,14 +9,20 @@ import favicon from '../public/static/img/favicon.ico'
 export default function MyApp({ Component, pageProps }) {
     const theme = createTheme({
         palette: {
-            primary: blue,
+            primary: 
+			{ 
+				main: '#F1F2F2',
+				light: '#fff',
```

Comment:
As far as I understand the palette usage, these colors should be all shades of one color – here they aren't, with `dark` being the primary branding color and `main` and `light` being contrastive with it.

Perhaps the branding red color should be used as `secondary.main`, to be used sparingly in the UI, and the primary colors should be defined as black-on-lightgray (`main: '#F1F2F2', contrastText: '#000'`), with no light and dark variants?

But it is possible that this choice has a purpose I don't understand.